india_stocks_api/internal/context.py

```
@@ -18,11 +19,112 @@
+    if path.exists():
+        try:
+            data = json.loads(path.read_text(encoding="utf-8"))
+            if isinstance(data, dict):
+                # Ensure nested structure is dict-of-dicts
+                _CREDS_CACHE = {
+                    str(broker): (creds if isinstance(creds, dict) else {})
+                    for broker, creds in data.items()
+                }
+            else:
+                _CREDS_CACHE = {}
+        except Exception:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)